tests/server/3-document.spec.js

```diff
@@ -71,13 +71,13 @@ describe('Document', () => {
           });
     });
 
-    it('should return err if something goes wrong with the server', (done) => {
+    it('should return err if something goes wrongtoken is used for accessing all documents', (done) => {
```

Comment:
### Critical

What exactly is this supposed to mean, did you read this back to yourself?